<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplingPage</class>
 <widget class="QWidget" name="SamplingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sampling Page</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="samplingGroupBox">
     <property name="title">
      <string>Sampling Configuration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="intervalLayout">
        <item>
         <widget class="QLabel" name="intervalLabel">
          <property name="text">
           <string>Sampling Interval (ms):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="intervalSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="durationLayout">
        <item>
         <widget class="QLabel" name="durationLabel">
          <property name="text">
           <string>Duration (s):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="durationSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>3600</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Start Sampling</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
